libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```
@@ -552,6 +555,8 @@
         "cvrInvalid": "Invalid CVR number",
         "cvrNumber": "CVR-Nr.",
         "country": "Country",
+        "phoneHint": "Phonenumber format +45 12345678",
```

nitpick: to be the same with value on line 738

```suggestion
        "phoneHint": "Phone number format +45 12345678",
```